vllm/v1/metrics/loggers.py

```diff
@@ -203,8 +205,28 @@ def __init__(self,
         self.gauge_scheduler_waiting = make_per_engine(gauge_scheduler_waiting,
                                                        engine_indexes,
                                                        model_name)
+        if envs.VLLM_SERVER_DEV_MODE:
+            # Engine sleep state
+            self.gauge_engine_sleep_state = self._gauge_cls(
+                name="vllm:engine_sleep_state",
+                documentation=("Engine sleep state."),
```

Comment:
Add some more documentation, including what the labels mean

Reply:
Done!